src/pages/scientificServices/imputation/views/JobHistory.tsx

```diff
@@ -1,20 +1,265 @@
+      headerRenderer: () => <HeaderCell>Submitted</HeaderCell>,
+      cellRenderer: ({ rowIndex }) => {
+        return <SubmittedCell pipelineRun={paginatedRuns[rowIndex]} />;
+      },
+      size: { basis: 90 },
+    },
+    {
+      field: 'completed',
+      headerRenderer: () => <HeaderCell>Completed</HeaderCell>,
+      cellRenderer: ({ rowIndex }) => {
+        return <CompletedCell pipelineRun={paginatedRuns[rowIndex]} />;
+      },
+      size: { basis: 90 },
+    },
+    {
+      field: 'quotaUsed',
+      headerRenderer: () => <HeaderCell>Quota Used</HeaderCell>,
+      cellRenderer: ({ rowIndex }) => {
+        return <QuotaUsedCell pipelineRun={paginatedRuns[rowIndex]} />;
+      },
+      size: { basis: 30 },
+    },
+    {
+      field: 'resultURL',
+      headerRenderer: () => <HeaderCell>Actions</HeaderCell>,
+      cellRenderer: ({ rowIndex }) => {
+        return <ActionCell pipelineRun={paginatedRuns[rowIndex]} />;
+      },
+      size: { basis: 100 },
+    },
+  ];
+};
+
+interface CellProps {
+  // I have absolutely no clue why TS thinks this is unused
+  // eslint-disable-next-line react/no-unused-prop-types
+  pipelineRun: PipelineRun;
+}
+
+const JobIdCell = ({ pipelineRun }: CellProps): ReactNode => {
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: '0.25rem', width: '100%' }}>
+      <div style={{ width: '100%', overflow: 'hidden' }}>
+        <TooltipCell
+          tooltip={pipelineRun.jobId}
+          style={{
+            whiteSpace: 'nowrap',
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            width: '100%',
+          }}
+        >
+          {pipelineRun.jobId}
+        </TooltipCell>
+      </div>
+      <div
+        style={{
+          width: 'fit-content',
+          fontWeight: 600,
+          backgroundColor: '#4D72AA4D',
+          padding: '0.33rem',
+          borderRadius: '4px',
+          fontSize: '10px',
+        }}
+      >
+        {pipelineRun.pipelineName}
+      </div>
+    </div>
+  );
+};
+
+const DescriptionCell = ({ pipelineRun }: CellProps): ReactNode => {
+  // descriptions can be long, so truncate them and allow the user to hover over them to see the full description
+  return <TooltipCell tooltip={null}>{pipelineRun.description}</TooltipCell>;
+};
+
+const StatusCell = ({ pipelineRun }: CellProps): ReactNode => {
+  return <div style={{ display: 'flex', alignItems: 'center' }}>{getRunStatusIcon(pipelineRun.status)}</div>;
+};
+
+const SubmittedCell = ({ pipelineRun }: CellProps): ReactNode => {
+  return <div>{new Date(pipelineRun.timeSubmitted).toLocaleDateString()}</div>;
+};
+
+const CompletedCell = ({ pipelineRun }: CellProps): ReactNode => {
+  return <div>{pipelineRun.timeCompleted ? new Date(pipelineRun.timeCompleted).toLocaleDateString() : ''}</div>;
+};
+
+const QuotaUsedCell = (props: CellProps): ReactNode => {
+  return <div>{props.pipelineRun.quotaConsumed ? props.pipelineRun.quotaConsumed : 'N/A'}</div>;
```

Comment:
Using text from [mockup](https://www.figma.com/design/xMS9OAHL39AlqDjCI7O4rQ/Imputation-UX-mocks?node-id=597-4411&m=dev):

```suggestion
  return <div>{props.pipelineRun.quotaConsumed ? `${props.pipelineRun.quotaConsumed} samples` : 'N/A'}</div>;
```